dask_sql/physical/rel/custom/ml_classes.py

```diff
@@ -0,0 +1,382 @@
+def get_cpu_classes():
```

Comment:
For organizational purposes, wonder if this file would make more sense under `dask_sql/physical/utils/`?